packages/http-client/src/remote-index-api.ts

```diff
@@ -17,19 +17,20 @@ export class RemoteIndexApi implements IndexApi {
   private readonly _collectionURL: URL
 
   constructor(apiUrl: URL) {
-    this._collectionURL = new URL('./collection', apiUrl)
+    this._collectionURL = new URL('./collection/', apiUrl)
```

Comment:
why is this necessary?

Reply:
For a server that does not matter. It matters for composing URLs. `./collection` means a "resource" named `collection` in the current directory of `apiURL`. If we want to have `./collection/count`, based on `this._collectionURL` using `new URL('./count', collectionURL)`, we have to indicate that `collection` is a "folder" not a "resource". Thus, a slash.